holidays/countries/bolivia.py

```diff
@@ -121,29 +106,21 @@ def _populate(self, year):
 
         # Pando Day.
         if self.subdiv == "N":
-            self[date(year, OCT, 11)] = "Dia del departamento de Pando"
+            self[date(year, OCT, 11)] = "Día del departamento de Pando"
 
         # All Soul's Day.
-        name = "Todos Santos"
-        self[date(year, NOV, 2)] = name
-
-        if self.observed and date(year, NOV, 2).weekday() == SUN:
-            self[date(year, NOV, 2) + td(days=+1)] = f"{name} (Observed)"
+        _add_with_observed(date(year, NOV, 2), "Todos Santos")
 
         # Potosí Day.
         if self.subdiv == "P":
-            self[date(year, NOV, 10)] = "Dia del departamento de Potosí"
+            self[date(year, NOV, 10)] = "Día del departamento de Potosí"
```

Comment:
We could consolidate the subdiv based holidays below and optimize them with `elif`s.